api/common.go

```diff
@@ -14,6 +14,54 @@ import (
 	"github.com/docker/libtrust"
 )
 
+type Command struct {
+	Name        string
+	Description string
+}
+
+var DockerCommands = map[string]Command{
```

Comment:
I have the same opinion as @duglin, it would be cool to not have to write twice the same thing like : `"images":  {"images", "List images"},`.